src/elements/Library/SortBar/SortBar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './SortBar.css';
+
+class SortBar extends React.Component {
+    render() {
+        return (
+        <div className="sortBar">SORT BY 
+            <select id="SORT BY">
```

Comment:
here you could receive props passing all possible sort by and mapping it, instead of hard coding it